kornia/geometry/quaternion.py

```diff
@@ -318,6 +318,51 @@ def random(cls, batch_size: int) -> 'Quaternion':
         q4 = r1.sqrt() * (2 * pi * r3).cos()
         return cls(stack((q1, q2, q3, q4), -1))
 
+    @classmethod
+    def slerp(cls, q0: 'Quaternion', q1: 'Quaternion', iratio: float) -> 'Quaternion':
```

Comment:
Also I would implement this as a free function instead outside of the class, and simplify the module function such as 

```python
def slerp(self, q1, t):
   # use self as for q0

```